scripts/extract_pdf_text.py

```diff
@@ -0,0 +1,22 @@
+import fitz
+import sys
+import argparse
+
+def extract_pdf_text(pdf_path: str, output_type: str = "text" ):
+    """ Extracts text from a PDF file and writes it to a text file. """
+    with fitz.open(pdf_path) as doc:
+        for page in doc: # iterate the document pages
+	        text = page.get_text(output_type).encode("utf8") # get plain text (is in UTF-8)
```

Comment:
do we want to use `sort=True` here? https://pymupdf.readthedocs.io/en/latest/page.html#Page.get_text

Reply:
also wondering, do we not want this text in unicode? (so, no `.encode()`?)

Reply:
I'll make the change for sort=True. This code is writing utf-8 bytes to stdout byte buffer, so `encode` should return the utf-8 bytes that we'd want to write.